.github/workflows/docs.yml

```diff
@@ -15,6 +15,11 @@ jobs:
         with:
           node-version-file: .nvmrc
           cache: npm
+      - uses: opensea-public-docs-publisher
```

Comment:
I don't think this will work, you need to use the action that's specified in the notion doc, sorry I didn't catch this sooner